content/en/docs/collector/internal-telemetry.md

```diff
@@ -38,38 +54,80 @@ exposes them using the OpenTelemetry Go
 [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/prometheus)
 for scraping at `http://127.0.0.1:8888/metrics`.
 
-The Collector can push its internal metrics to an OTLP backend via the following
-configuration:
-
-```yaml
-service:
-  telemetry:
-    metrics:
-      readers:
-        - periodic:
-            exporter:
-              otlp:
-                protocol: http/protobuf
-                endpoint: https://backend:4318
-```
-
-Alternatively, you can expose the Prometheus endpoint to one specific or all
-network interfaces when needed. For containerized environments, you might want
-to expose this port on a public interface.
-
-Set the Prometheus config under `service::telemetry::metrics`:
-
-```yaml
-service:
-  telemetry:
-    metrics:
-      readers:
-        - pull:
-            exporter:
-              prometheus:
-                host: '0.0.0.0'
-                port: 8888
-```
+There are three ways to export internal Collector metrics.
+
+1. Self-ingesting, exporting internal metrics via the
+   [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter).
+
+   You can expose the Prometheus endpoint to one specific or all network
+   interfaces when needed. For containerized environments, you might want to
+   expose this port on a public interface.
+
+   Set the Prometheus export configuration under `service::telemetry::metrics`:
+
+   ```yaml
+   service:
+     telemetry:
+       metrics:
+         readers:
+           - pull:
+               exporter:
+                 prometheus:
+                   host: '0.0.0.0'
```

Comment:
Is `0.0.0.0` right here?

- I'm not sure if this will work for IPv6 only stacks
- The default behaviour has been changed in endpoints and such from `0.0.0.0` to `localhost`. Should we just be using `localhost` if we are intending to expose for self scraping, or be more explicit for public interfaces?

Probably not something we need to tackle here, but I'd love a chime in with anyone with better container/Linux networking knowledge than me.